internal/resources/policy/kind/custom/input_schema.go

```diff
@@ -33,10 +45,10 @@ var (
 				reciperesource.TMCExternalIPSKey:              reciperesource.TMCExternalIps,
 				reciperesource.TMCHTTPSIngressKey:             reciperesource.TMCHTTPSIngress,
 				reciperesource.TMCRequireLabelsKey:            reciperesource.TMCRequireLabels,
+				reciperesource.TMCCustomKey:                   reciperesource.TMCCustomSchema,
```

Comment:
Is acceptance test added for the new custom schema?

Please attach the screenshot of the same

Reply:
I didn't find acceptance tests for the custom policy resource in general, I did implement mock tests though.